topic-operator/src/main/java/io/strimzi/operator/topic/Main.java

```diff
@@ -25,6 +26,11 @@
 
     private final static Logger LOGGER = LogManager.getLogger(Main.class);
 
+    static String blockingWorkerPoolName = "blocking-ops-executor";
+
+    // TODO, change this if/when this worker pool is shared with ZkImpl that has its own worker pool also
+    static int blockingWorkerPoolSize  = 1;
```

Comment:
Should these be `public / private final ...`?

Reply:
Good point. I've refactored them away now :)